tests/wpunit/Migrations/FeePlanMigrationTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+namespace Migrations;
+
+use Codeception\Util\Debug;
+use WPBDP__Fee_Plan;
+
+class FeePlanMigrationTest extends \Codeception\Test\Unit {
+
+	/**
+	 * @var \WpunitTester
+	 */
+	protected $tester;
+
+	protected function _before() {
+		$this->save_fee();
+	}
+
+	protected function _after() {
+		global $wpdb;
+
+		// Remove any existing payment in database
```

Comment:
This comment doesn't make sense to me.

Reply:
Test cases on hold until I get a DB function working with the `$wpdb` functions. Its skipped for now